config.js

```diff
@@ -80,6 +80,8 @@ var config = { // eslint-disable-line no-unused-vars
     startScreenSharing: false, // Will try to start with screensharing instead of camera
 //    startAudioMuted: 10, // every participant after the Nth will start audio muted
 //    startVideoMuted: 10, // every participant after the Nth will start video muted
+    startWithAudioMuted: false, // will start with the microphone muted
+    startWithVideoMuted: false, // will start with the camera turned off
```

Comment:
Maybe not for this PR, but we should probably deprecate the startAudioMuted and startVideoMuted above and rename them to allStartAudioMuted and allStartVideoMuted, because it's confusing right now.